### PR TITLE
Update and fix Catalan translation

### PR DESCRIPTION
**Pull request by jmontane**

### CODE REVIEW

**Comment by m4sk1n** [comment]
I have made PR for your repo, fixing syntax error, can you merge it, or fix it by your own? :)

**Comment by m4sk1n** [comment]
still the same thing happens, i'll fix it and check it on my pc…

File: config/locales/devise.ca.yml [hunk 1]
```diff
@@ -2,60 +2,60 @@
...
-      invalid: Invàlid %{authentication_keys} o contrassenya.
-      last_attempt: Té un intent més abans que el seu compte sigui bloquejat.
-      locked: el seu compte ha estat bloquejat.
-      not_found_in_database: Invàlit %{authentication_keys} o contrassenya.
-      timeout: la sessió ha expirat. Si us plau iniciï sessió de nou per a continuar.
-      unauthenticated: Necessita iniciar sessió o registrar-se abans de continuar.
-      unconfirmed: Ha de confirmar la seva adreça de correu electrònic abans de continuar.
+      already_authenticated: Ja estàs registrat.
+      inactive: el teu compte encara no s'ha activat.
+      invalid: %{authentication_keys} o contrasenya no són vàlids.
```
Comment (ykzts) [inline config/locales/devise.ca.yml]: `invalid: "%{authentication_keys} o contrasenya no són vàlids."`

**Comment by m4sk1n** [comment]
ok, you can merge the current one PR, it's correct (checked)…

File: config/locales/devise.ca.yml [hunk 2]
```diff
@@ -2,60 +2,60 @@
...
+      last_attempt: Tens un intent més, abans que es bloqui el compte.
+      locked: el compte s'ha blocat.
+      not_found_in_database: %{authentication_keys} o contrasenya no vàlids.
```
Comment (ykzts) [inline config/locales/devise.ca.yml]: `not_found_in_database: "%{authentication_keys} o contrasenya no vàlids."`

**Review by ykzts — CHANGES_REQUESTED** [review]

**Review by mkljczk — APPROVED** [review]

**Review by ykzts — APPROVED** [review]

**Pull request merged**